src/world_modeling/occupancy_segmentation/src/occupancy_segmentation_node.cpp

```diff
@@ -0,0 +1,56 @@
+#include "occupancy_segmentation_node.hpp"
+#include <iostream>
+#include <memory>
+
+OccupancySegmentationNode::OccupancySegmentationNode() : Node("occupancy_segmentation") {
+  _subscriber = this->create_subscription<sensor_msgs::msg::PointCloud2>(
+      "/velodyne_points", 10,
```

Comment:
specifying the topic name should be from the params.yaml file

Reply:
done